tools/test/audio/comp_run.sh

```diff
@@ -121,13 +123,28 @@ run_testbench ()
 
 parse_args "$@"
 
-# Paths
-HOST_ROOT=../../testbench/build_testbench
-HOST_EXE=$HOST_ROOT/install/bin/testbench
-HOST_LIB=$HOST_ROOT/sof_ep/install/lib
-TPLG_LIB=$HOST_ROOT/sof_parser/install/lib
+# Path to topologies
 TPLG_DIR=../../build_tools/test/topology
 
+# Testbench path and executable
+if [[ -z $XTRUN ]]; then
+    TESTBENCH=../../testbench/build_testbench/install/bin/testbench
+else
+    BUILD_DIR=../../testbench/build_xt_testbench
+    TESTBENCH="$BUILD_DIR"/testbench
+    source "$BUILD_DIR"/xtrun_env.sh
+    VALGRIND=false
```

Comment:
```suggestion
   if $VALGRIND; then
      >&2 printf "WARNING: ignoring VALGRIND with xt-run\n`
      VALGRIND=false
   fi
   
PS: Quotes are not required because VALGRIND is either `true` or `false` which does not need quotes. Also, if $VALGRIND is accidentally empty then 
```

Reply:
I've fixed this as suggested. I noticed that  made a mistake with HOST_EXE, the VALGRIND_CMD is applied separately in run_testbench function. It's fixed now.